Gruntfile.js

```diff
@@ -261,7 +328,7 @@ module.exports = function(grunt) {
     require('load-grunt-tasks')(grunt);
 
     // Where we tell Grunt what to do when we type "grunt" into the terminal.
-    grunt.registerTask('default', ['sass','assemble','premailer','imagemin','replace:src_images']);
+    grunt.registerTask('default', ['sass','autoprefixer:preview','assemble','premailer','imagemin','replace:src_images']);
```

Comment:
Only if you want less browser coverage. Not recommended.
Unless there's actual bugs / issues, i'd recommend going through with the PR and we can enhance later as needed.